bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionRegistry.java

```diff
@@ -166,11 +196,11 @@ protected void removeConfigDescriptionProvider(ConfigDescriptionProvider configD
             List<ConfigDescriptionParameter> parametersWithOptions = new ArrayList<ConfigDescriptionParameter>(
                     parameters.size());
             for (ConfigDescriptionParameter parameter : parameters) {
-                parametersWithOptions.add(getConfigOptions(uri, parameter, locale));
+                parametersWithOptions.add(getConfigOptions(aliasedURI, parameter, locale));
```

Comment:
Here you are missing the whole point of the aliases: The UI requests the config description of a thing (and not its thing-type directly) because there might be config options available that are only applicable for this thing, but not all of the same type. Therefore, you'd have to ask for the config options of the original uri - and yes, maybe here again, you could use the aliasedURI results as a fallback, if you do not get anything else.